src/toil/test/wdl/standard_library/ceil_as_command.wdl

```diff
@@ -0,0 +1,16 @@
+workflow ceilWorkflow {
+  Float num
+  call get_ceil { input: num=num }
+}
+
+task get_ceil {
+  Float num
+
+  command {
+    echo ${ceil(num)} > the_ceiling.txt
+  }
+
+ output {
+    File the_ceiling = 'the_ceiling.txt'
```

Comment:
This wants to be a standardized/generic `output.txt` or something so it can be the same file to read the result from for all these WDL tests.